### PR TITLE
Align to message house tagline, RKVST brand, and WWDW order

### PR DESCRIPTION
**Pull request by JAG-UK**

### CODE REVIEW

**Review by leflambeur — CHANGES_REQUESTED** [review]
Before I approve this, do we also want to update the Logos and other Branding in this PR while we're here?

If not LGTM, we just need to be proactive in case anything is missed anywhere

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Jitsuin RKVST Documentation
+# RKVST Documentation
 
-This is the Home of the Jitsuin RKVST Documentation
+This is the Home of the RKVST Documentation
 
-Jitsuin RKVST is a Multi-Party Automation API powered by the Blockchain find out more on our [website](https://jitsuin.com) or read through our [Docs]()
+RKVST provides Continuous Assurance as a Service to attest trustworthiness of assets, and boost confidence in critical decisions.  Find out more on our [website](https://jitsuin.com).
```
Comment (creator-of-data-assurance-hub) [inline README.md:5]: change the website link to RKVST.com

**Review by leflambeur — APPROVED** [review]
Approved- other changes will be in a different PR at a later date

File: README.md (hunk 1 above)
Comment (leflambeur) [reply README.md:5]: That currently points at docs.jitsuin.com?

Is rkvst.com live yet?

**Comment by leflambeur** [comment]
I will merge this now and other updates including URLs for other sites can be updated at a later date with ease

File: content/docs/user-patterns/_index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ weight: 999
 toc: false
 ---
 
-RKVST is a very flexible system, and enables users to record When Who Did almost anything to anything. To get the best out of the system, however, it is important to model your real-world assets and business processes efficiently into RKVST Assets and Events.
+RKVST is a very flexible system, and enables users to record Who Did What When almost anything to anything. To get the best out of the system, however, it is important to model your real-world assets and business processes efficiently into RKVST Assets and Events.
```
Comment (creator-of-data-assurance-hub) [inline content/docs/user-patterns/_index.md:13]: Who did what when _to_ almost anything

**Review by creator-of-data-assurance-hub — COMMENTED** [review]
Just a few minor nits, looks good

File: README.md (hunk 1 above)
Comment (leflambeur) [reply README.md:5]: This isn't updated yet so we'll leave it

File: content/docs/user-patterns/_index.md (hunk 2 above)
Comment (leflambeur) [reply content/docs/user-patterns/_index.md:13]: I've already merged so this will be answered in a different PR

File: README.md (hunk 1 above)
Comment (creator-of-data-assurance-hub) [reply README.md:5]: imminent.

**Pull request merged**